cloudinit/config/cc_set_passwords.py

```diff
@@ -101,6 +81,12 @@ def handle_ssh_pwauth(pw_auth, distro):
     @return: None"""
     cfg_name = "PasswordAuthentication"
 
+    if isinstance(pw_auth, str):
+        LOG.warning(
+            "DEPRECATION: The 'ssh_pwauth' config key should be set to "
```

Comment:
Thanks for deprecation. it's not effective to support "" and "unchanged" which are equivalent of false.